supabase/migrations/20231017171314_my_table.sql

```diff
@@ -0,0 +1,54 @@
+drop policy "Enable read access for authenticated users." on "public"."pallet";
+
+drop policy "Enable insert for authenticated users only" on "public"."project";
+
+drop policy "Enable read access for authenticated users." on "public"."shipment";
+
+alter table "public"."pallet" disable row level security;
+
+alter table "public"."pallet_scan" disable row level security;
+
+alter table "public"."project" disable row level security;
+
+alter table "public"."shipment" add column "bookkeeping_flexport" boolean;
+
+alter table "public"."shipment" add column "bookkeeping_gik" boolean;
+
+alter table "public"."shipment" add column "commercial_invoice_link" text;
+
+alter table "public"."shipment" add column "consignee" text;
+
+alter table "public"."shipment" add column "destination" text not null;
```

Comment:
Shall we create a `countries` table and link to that by ID here? We can then eventually hook that into the geography data and link to specific areas that it will be deployed to.

Do you know how accurate we tend to be here? Is it specific buildings / cities / countries?